src/plugins/data/public/search/search_service.ts

```diff
@@ -111,6 +111,9 @@ export class SearchService implements Plugin<ISearchSetup, ISearchStart> {
     return {
       aggs: this.aggsService.start({ fieldFormats, uiSettings }),
       search,
+      showError: (e: any) => {
```

Comment:
does it has to be `any` or it could be better typed? Maybe it could allow only our search errors